assets/src/blocks/Counter/CounterBlock.js

```diff
@@ -52,45 +52,46 @@ export class CounterBlock {
           save() {
             return null;
           },
-        }
+        },
       ],
-      edit: ( { isSelected, attributes, setAttributes } ) => {
+      // Renamed attributes to editAttributes to avoid eslint error.
+      edit: ( { isSelected, editAttributes, setAttributes } ) => {
```

Comment:
These attributes are WP defaults, and therefore you can't change their name like this... In fact the block now crashes in the editor 😕 
So either we rename the other `attributes` object to something else, or maybe we can override this one like this: `{ attributes: editAttributes }` (but I'm not 100% sure that the latter will work 🤔)

Reply:
Or you could use `edit: (props) =>` and then use `props.attributes` for example

Reply:
That was my initial error, because of the theme I have for my IDE I didn't take note that we were destructuring those values.